qiskit/quantum_info/operators/symplectic/clifford_circuits.py

```diff
@@ -91,6 +91,18 @@ def _append_operation(clifford, operation, qargs=None):
             raise QiskitError("Invalid qubits for 2-qubit gate.")
         return _BASIS_2Q[name](clifford, qargs[0], qargs[1])
 
+    # If gate is a Clifford, we can either unroll the gate using the "to_circuit"
+    # method, or we can compose the Cliffords directly. Experimentally, for large
+    # cliffords the second method is considerably faster.
+
+    # pylint: disable=cyclic-import
+    from qiskit.quantum_info import Clifford
+
+    if isinstance(gate, Clifford):
+        composed_clifford = clifford.compose(gate.copy(), qargs=qargs.copy(), front=False)
```

Comment:
Why is `copy` of `gate` necessary? (Sorry, I don't fully understand.)

Reply:
Thanks! I have removed the unnecessary copying in [17f7ce0](https://github.com/Qiskit/qiskit-terra/pull/9169/commits/17f7ce0b8cecc69046951951ca749607f3dc88b6).